.dockerignore

```diff
@@ -2,9 +2,11 @@
 
 !config
 !dao
+!fixtures
 !handlers
 !helpers
 !interceptors
+!mappers
```

Comment:
I believe a transformer is the equivalent of a mapper

Reply:
Transformers were created to specifically transform the **same** object to its rest / db representation and they implement a common interface to achieve this. Mappers package was created to handle more "general" transformations such as between different response objects. Happy to change that if you feel I should.